downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md

```diff
@@ -0,0 +1,400 @@
+
+Included record parameter as the last of the parameter specify options.
+
+```ballerina
+public type Person record {
+	string name;
+	float? score = 0;
+};
+
+public function main(*Person person) {
+	// Process data here
+}
+```
+
+```bash
+bal run file.bal -- --name riyafa --score=99.9
+```
+
+In the above example, `name` and `score` are options. `riyafa` and `99.9` are arguments of the option.
+
+**Operands**
+
+Other parameters that are not included records specify operands; for these parameters, the position is significant and the name is not.
+
+```ballerina
+public type Person record {
+	string name;
+	float? score = 0;
+};
+public function main(int efficiency, string character, *Person person) {
+	// Process data here
+}
+```
+
+```bash
+bal run file.bal -- --name riyafa  100 --score=99.9 Good
+```
+
+This example, which is the same as above includes `100`, which gets mapped to `efficiency`, and `Good`, which gets mapped to `character` as operands. 
+
+Both operand and option parameters can be of types int, float, decimal, string, array of any of these types, and union of any of these types with nil. 
+
+Additionally, option parameters can be of types `boolean`, `boolean[]`, or `boolean?`.
+
+**Operand Arrays**
+
+>**Note:** If there is an operand parameter of type O[], then it cannot be followed by parameters of type O[], O?, and O x = d. Here O stands for a type that is a subtype of one of string, float, or decimal. An array value is specified by repeatedly specifying the `option` parameter.
+
+If `scores` is an int array, then,
+
+```bash
+bal run file.bal -- --scores=10 --scores=20 --scores=30
+```
+
+This produces the following int array.
+
+```bash
+[10, 20, 30]
+```
+
+**Option Boolean Parameters**
+
+When there’s an option of `boolean`, `boolean[]`, or `boolean?` type, it does not take an `option` argument. The presence of the option is considered to be `true` and the absence of it is considered to be false. 
+In the following example, suppose `results` is a boolean array.
+
+```bash
+bal run file.bal -- --results --results --results
+```
+
+This produces the following boolean array.
+
+```bash
+[true, true, true]
+```
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+### Standard Library Updates
+
+#### New Features
+
+##### GraphQL Package
+
+- Added named fragment support
+- Added enum support
+- Added map support
+- Added compiler plugin validation for GraphQL services
+
+##### Email Package
+Added compiler plugin validation for Email listener services.
+
+##### UDP Package
+Added compiler plugin validation for UDP services.
+
+##### HTTP Package
+Added compiler plugin validations for HTTP services.
+
+##### WebSocket Package
+Added compiler plugin validation for WebSocket services.
+
+##### Websub Package
```

Comment:
```suggestion
##### WebSub Package
```

Reply:
Changed to "`websub` Package" as per https://github.com/ballerina-platform/ballerina-dev-website/pull/2113#discussion_r618606537